### DISCUSSION TITLE
Query regarding RA Positions

### DISCUSSION THREAD

**Discussion by katedrift** [snigi-gupta/AttendingUniversityAtBuffalo#32] [General]
- If I choose to apply for a Research Assistant position, will this timeframe be deducted from CPT or OPT? 
- If I also plan to do an internship during the summer 2022 (1.5 y course starting Fall 2021), do you think it is advisable to not apply for Research Assistantposition during the other semesters? 
- Is it generally difficult to balance an Research Assistant position along with course work?
Thank you

**Comment by snigi-gupta** [comment]
Hi @katedrift 
Firstly I apologize for the delay in my response. Lately, I have been dealing with a lot of change, hence the delay.

1. No, and RA is an on-campus job. This time-period will not be deducted from your CPT or OPT
2. If I have understood your ques correctly then are you asking that, should you do RA in fall, spring sems + Internship in Summer? right? In that case YES. Your RA will be active only during the sems. so do that if your study work-load allows you. During the summer you will be fairly free and doing an internship is a boost in your resume and work experience. If your ques was to do both RA and internship (like a co-op) during the same sem along with studies, I would say it all depends on your course structure. definitely, the first priority is keeping a GPA over 3.0 and academics. so don't let anything hamper that.
3. Not really, if you stay disciplined you will be able to do both together. All you need to do is cut some time off from your leisure and make sure you are sticking to your schedule. I know plenty of people who have managed both gracefully. :)

All the best!

**Reply by katedrift** [reply]
Thank you for the clarification!
